lms/djangoapps/support/static/support/jsx/errors_list.jsx

```diff
@@ -6,25 +6,28 @@ import PropTypes from 'prop-types';
 
 class ShowErrors extends React.Component {
   render() {
-    if (this.props.errorList.length > 0) {
-      window.scrollTo(0, 0);
-    }
-    return this.props.errorList.length > 0 &&
-      <div className="col-sm-12">
+    return (
+      this.props.hasErrors && <div className="col-sm-12">
         <div className="alert alert-danger" role="alert">
           <strong>{gettext('Please fix the following errors:')}</strong>
           <ul>
-            {this.props.errorList.map((error, i) =>
-              <li key={i}>{error}</li>,
+            { Object.keys(this.props.errorList).map(key =>
+              this.props.errorList[key] && <li key={key}>{this.props.errorList[key]}</li>,
             )}
           </ul>
         </div>
-      </div>;
+      </div>);
   }
 }
 
 ShowErrors.propTypes = {
-  errorList: PropTypes.arrayOf(PropTypes.object).isRequired,
+  errorList: PropTypes.shape({
+    course: PropTypes.string,
```

Comment:
maybe we can use a bool type if the variable only needs to be used for a true or false condition

Reply:
I have changed it to `objectOf`.